src/Api/ApiCalls.js

```diff
@@ -0,0 +1,14 @@
+import { useDispatch } from 'react-redux';
+import axios from '../config/axios';
+import { Login } from '../redux/slices/auth';
+
+const dispatch = useDispatch();
+
+export default async function UserLogin(data) {
+  await axios.post('/login', data).then((res) => {
+    if (res.status === 200) {
+      dispatch(Login(res.data));
+      window.location.reload();
```

Comment:
Do we really need to reload our page after the API call? Let's remove it so that our project can function normally as a real SPA.

Reply:
@aimalamiri Actually yes we need to refresh our page to trigger the `dispatch(isLoggedIn());` inside the useEffect , so we get all of the user infos saved in local storage.
**_### 🛑 Please Note that The refresh will happens Once & Only when the user Login/Logout_**